external/systemjs/plugin-babel-cached.js

```diff
@@ -110,5 +111,7 @@ exports.translate = function (load, opt) {
         return translated;
       });
     });
```

Comment:
I purposely added the `catch` at the *very* end, since this way it'll catch errors occurring at *any* point in the entire Promise chain. (Note also that the Private Browsing related error is throw already in `loadCache`, so your suggestion wouldn't have have worked here).

The point of this patch was thus to not only fix the problem at hand, but to ensure that the code will always attempt to fallback no matter what (future) errors are encountered.
To avoid trying to re-parse the code when `babel.translate` specifically fails, which I gather you wanted to avoid, the patch now tracks that particular case separately.